scripts/feature_tests/README.md

```diff
@@ -2,9 +2,9 @@
 
 ## Build Status
 
-[![Ubuntu V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/badge/icon?subject=Feature-tests)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_ubuntu/)
-[![Centos V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/badge/icon?subject=Feature-tests-centos)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_centos/)
-[![Ubuntu V1alpha4 build status](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_ubuntu/badge/icon?subject=Feature-tests-upgrade)](https://jenkins.nordix.org/view/Airship/job/airship_master_feature_tests_upgrade_ubuntu/)
+[![Ubuntu V1alpha4 build status](https://jenkins.nordix.org/view/Metal3/job/Metal3_main_feature_tests_ubuntu/badge/icon?subject=Feature-tests)](https://jenkins.nordix.org/view/Metal3/job/Metal3_main_feature_tests_ubuntu/)
```

Comment:
This is outdated, we use v1beta1 for feature tests

Reply:
Actually this text was whole wrong. It should be main feature test.

Reply:
Done